### ISSUE TITLE
Change module description of spatial analysis

### ISSUE THREAD

**Issue by schroedtert** [PedestrianDynamics/PedPy#346]
The current description is from an earlier stage of development:
```python
"""Module containing functions to compute pair distribution function."""
```

It should be more general and not only mention the pair distribution function, as there will be more functions coming in the future.

**Comment by chraibi** [comment]
We will update the description when more function are implemented.

**Comment by schroedtert** [comment]
It should already be updated now, the module description is also shown on the website and is misleading from my point of view:

![Screenshot_20240605-103240.png](https://github.com/PedestrianDynamics/PedPy/assets/15690524/a5d52502-181a-43af-bce4-3450ee6c324d)

**Comment by chraibi** [comment]
closed by #350
